Database/DatabaseSender.py

```diff
@@ -19,17 +19,18 @@ def create_requirements(self):
         create requirements table for each major
         """
         command = "CREATE TABLE " + self.requirements_table + """ (
-            id SERIAL PRIMARY KEY,
+            id NUMBER GENERATED BY DEFAULT AS IDENTITY,
             program_name VARCHAR(255),
             plan_type VARCHAR(255),
-            course_codes VARCHAR(5000),
+            course_codes VARCHAR(4000),
             number_of_courses int,
             credits_required DECIMAL(4,2),
             additional_requirements VARCHAR(255), 
             major_name VARCHAR(255),
             faculty VARCHAR(50),
             link VARCHAR(255),
-            year VARCHAR(20)
+            year VARCHAR(20),
+            PRIMARY KEY (id)
```

Comment:
if we have "id SERIAL PRIMARY KEY", we don't need to declare Primary Key here?

What's the reason behind this change?

Reply:
Oracle does not have SERIAL type